hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/ColumnStatIndexTestBase.scala

```diff
@@ -280,6 +327,62 @@ class ColumnStatIndexTestBase extends HoodieSparkClientTestBase {
     }
   }
 
+  protected def validatePartitionStatsIndex(testCase: ColumnStatsTestCase,
```

Comment:
So, this method just validates the index contents. Are there validation on partition pruning based on stats? I believe there might already be such a case before this patch. But, can you please confirm?